.github/prep-hotfix-sources.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+HOTFIX_TAG=$1
+REPO_PATH=$2
+SPEC_PATH=$3
+
+# Create OSG build dirs
+build_dir=$(pwd)/_build_dir
+for dirname in osg upstream; do
+    mkdir -p $build_dir/$dirname
+done
```

Comment:
```suggestion
mkdir -p $build_dir/{osg,upstream}
```

Reply:
Also, is it allowed for `$PWD` to have whitespace special characters?  If so, uses of `$build_dir` will need to be quoted.